extension/manifest.json

```diff
@@ -13,12 +13,10 @@
   "permissions": [
     "alarms",
     "experienceSamplingPrivate",
-    "identity",
     "notifications",
     "storage",
     "unlimitedStorage",
-    "https://chrome-experience-sampling.appspot.com/_ah/api/*",
-    "https://www.google.com/*"
+    "https://chrome-experience-sampling.appspot.com/_ah/api/*"
```

Comment:
Just for my own understanding: Are you reading some sort of CSRF token from this URL that you use later on to post data?